python/mlc_chat/cli/compile.py

```diff
@@ -139,6 +139,22 @@ def _check_prefix_symbols(prefix: str) -> str:
         "4) Android, iOS: .tar (objects); "
         "5) Web: .wasm (web assembly)",
     )
+    parser.add_argument(
+        "--sliding-window",
+        type=int,
+        default=None,
+        help="The sliding window size in sliding window attention (SWA). "
```

Comment:
Another noob question: what's the relationship between `--sliding-window` and `--max-sequence-length`? For Mistral models, is it true that we only need one option here? If so, we could unify them into `--max-sequence-length`